templates/app-form.html

```diff
@@ -197,11 +234,13 @@ <h1>Techtonica Application</h1>
         name="phone-number"
         id="phone-number"
         pattern="^\+?1?[-.\s]?(\d{3})[-.\s]?(\d{3})[-.\s]?(\d{4})$"
-        placeholder="ex. 415-555-1234 or 1-415-555-1234"
+        placeholder="example: 415-555-1234 or 1-415-555-1234"
```

Comment:
Can you remove "example", the or, and second phone number please? multiple examples and a pseudo-realistic phone number is a lot of cognitive load for the user
```suggestion
XXX-XXX-XXXX
```